testChecker.cpp

```diff
@@ -0,0 +1,26 @@
+#include "testChecker.h"
+#include "checker.h"
+
+void TestVitalsMonitor::TestVitals() {
+  assert(!VitalsMonitor::vitalsOk(99, 102, 70));
+  assert(VitalsMonitor::vitalsOk(98.1, 70, 98));
+}
+
+void TestVitalsMonitor::TestBodyTemparature() {
+  assert(VitalsMonitor::bodyTemperatureOk(99));
+  assert(VitalsMonitor::bodyTemperatureOk(98.1));
+  assert(false == VitalsMonitor::isBodyTemperatureNotOk(99));
+  assert(false == VitalsMonitor::isBodyTemperatureNotOk(98.1));
+}
+
+void TestVitalsMonitor::TestPulseRate() {
+  assert(!VitalsMonitor::pulseRateOk(102));
+  assert(VitalsMonitor::pulseRateOk(70));
+  assert(true == VitalsMonitor::isPulseRateNotOk(102));

```

Comment:
Notice the repeating pattern of tests in this and the above function. This is an indication that we need an abstraction. Try making something like `isInRange`. Then only one set of tests is enough to prove it - no need to re-test with every vital in future.